src/Components/Assessment.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Box from '@material-ui/core/Box';
+import AssessmentSlider from './AssessmentSlider';
+import { useTheme } from '@material-ui/core/styles';
+import KeyboardArrowDown from '@material-ui/icons/KeyboardArrowDown';
+import KeyboardArrowUp from '@material-ui/icons/KeyboardArrowUp';
+
+import { ExpandButton, useStyles } from './AssessmentStyle';
+
+export default function Assessment(props) {
+  const classes = useStyles();
+  const theme = useTheme();
+
+  const [expandButton, setExpandButton] = React.useState(false);
+
+  const handleExpandButton = () => {
+    setExpandButton(!expandButton);
+  };
+
+  return (
```

Comment:
Seeing some extra whitespace between the header and the assessment title:
![Screen Shot 2021-10-12 at 12 17 05 PM](https://user-images.githubusercontent.com/13375042/136994531-222d2526-2e12-4451-8d3e-a24263112757.png)